csharp/ql/Directory.Build.props

```diff
@@ -0,0 +1,6 @@
+<Project>
```

Comment:
This file is not strictly needed.

Reply:
If this is not strictly needed, are the changes in the stub generator not strictly needed either (I suppose this file currently has the same effect for stubs used by the QL tests as the project local `Directory.Build.props` that will be generated going forward)?

Reply:
It's not strictly needed, because when we run integration tests through sembuild, we're moving the files to the targets folder and then the settings in `Directory.Build.props` is not going to be inherited.

At the same time, the stub generator changes are needed, because we're creating a `stubs_output` subfolder in the `csharp` directory, so there the `Directory.Build.props` would be inherited.